src/InstagramClone.Api/Controllers/UsersController.cs

```diff
@@ -36,5 +36,36 @@ public IActionResult GetUser(Guid id)
             return Ok(user);
         }
 
+
+        
+        [HttpPost("")]
+        public  IActionResult PostUsers([FromBody] UserCreateDTO user)
+        {
+
+            if (user == null)
+                return BadRequest();
+            User userCreateDTO = new User()
+            {
+                Id=user.Id,
+                Email=user.Email,
+                Password=user.Password,
+                FirstName=user.FirstName,
+                LastName=user.LastName,
+                Avatar=user.Avatar,
+            };
```

Comment:
why is this object called `userCreateDTO`?

Reply:
I thought it was an ok name to use. I will change it to userDto